public/pages/Dashboard/Container/Dashboard.tsx

```diff
@@ -29,16 +29,21 @@ export const Dashboard = () => {
   const [isLoading, setIsLoading] = useState(true);
 
   const onRefreshPage = async () => {
-    await dispatch(
-      getDetectorList({
-        from: 0,
-        size: 1,
-        search: '',
-        sortDirection: SORT_DIRECTION.DESC,
-        sortField: 'name',
-      })
-    );
-    setIsLoading(false);
+    try {
+      await dispatch(
+        getDetectorList({
+          from: 0,
+          size: 1,
+          search: '',
+          sortDirection: SORT_DIRECTION.DESC,
```

Comment:
Should we use ASC sort direction?  `name` is just string, is it better to show alphabetically?

Reply:
This is just used to check if there is any detector ever created. That is why `size` is 1. If no detector, we move to EmptyDashboard page. If yes, we navigate to the real Dashboard page. using ASC or DESC doesn't matter too much here.

Reply:
any smarter way of doing the check of existing detector is appreciated.

Reply:
Sure, got it, that's ok. Maybe we can use `_count` in future.
https://stackoverflow.com/questions/25739888/counting-number-of-documents-using-elasticsearch
```
Probably _count is a bit faster since it doesn't have to execute a full query with ranking and result fetching and can simply return the size.
```